src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -281,11 +332,20 @@ public IIntellisenseResult Suggest(string expression, RecordType parameterType,
         /// </summary>
         public IIntellisenseResult Suggest(CheckResult checkResult, int cursorPosition)
```

Comment:
>checkResult [](http://example.com/codeflow?start=55&length=11)

We should check this is not null

Reply:
and have a test

Reply:
We have 100s of methods missing these checks. Will handle separately.